invisible_cities/cities/beersheba.py

```diff
@@ -67,13 +72,13 @@ def deconvolve_signal(det_db          : pd.DataFrame,
                       iteration_tol   : float,
                       sample_width    : List[float],
                       bin_size        : List[float],
-                      diffusion       : Tuple[float]=(1., 1., 0.3),
-                      energy_type     : HitEnergy=HitEnergy.Ec,
-                      deconv_mode     : DeconvolutionMode=DeconvolutionMode.joint,
-                      n_dim           : int=2,
-                      cut_type        : CutType=CutType.abs,
-                      inter_method    : InterpolationMethod=InterpolationMethod.cubic,
-                      n_iterations_g  : int=0):
+                      diffusion       : Optional[Tuple[float, float, float]]=(1., 1., 0.3),
+                      energy_type     : Optional[HitEnergy]=HitEnergy.Ec,
+                      deconv_mode     : Optional[DeconvolutionMode]=DeconvolutionMode.joint,
+                      n_dim           : Optional[int]=2,
+                      cut_type        : Optional[CutType]=CutType.abs,
+                      inter_method    : Optional[InterpolationMethod]=InterpolationMethod.cubic,
+                      n_iterations_g  : Optional[int]=0):
```

Comment:
Minor point: this does not conform to the IC alignment style. There are a few more instances of this in the code.